### PR TITLE
add subwallet docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PRs:
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/794
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/802

### CODE REVIEW

File: tokens/connect/subwallet.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+
+与SubWallet交互有多种方式，例如**通过浏览器扩展程序、移动端App或网页端数据面板**。
+
+You can get started by heading to [SubWallet's download page](https://www.subwallet.app/download.html){target=_blank} and downloading SubWallet for the platform of your choice.
+
+首先前往[SubWallet下载页面](https://www.subwallet.app/download.html){target=_blank}，根据个人使用习惯下载SubWallet。
+
+If you choose to use the web-accessible dashboard, you won't need to download anything. You can access the dashboard at [web.subwallet.app](https://web.subwallet.app/welcome){target=_blank}.
+
+如果您选择使用可通过网络访问的数据面板，您无需下载任何东西，只需通过[web.subwallet.app](https://web.subwallet.app/welcome){target=_blank}访问数据面板即可。
```
Comment (wuzhong-papermoon) [inline tokens/connect/subwallet.md:37]:
```suggestion
如果您选择使用通过网页访问的数据面板，您无需下载任何东西，只需在[web.subwallet.app](https://web.subwallet.app/welcome){target=_blank}访问数据面板即可。
```

File: tokens/connect/subwallet.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+
+如果您选择**Import from seed phrase**（从助记词导入），从助记词导入账户时可能会出现一些不兼容问题。例如，Trust Wallet和SafePal钱包都与SubWallet不兼容。如果您遇到不兼容的问题SubWallet建议您创建一个新钱包。
+
+If you select **Import from Polkadot.{js}**, you'll need to make sure that the account was created in Polkadot.js via private key. If it was created with a seed phrase and you attempt to import it to SubWallet, a different public address will be used. This is because Polkadot.js uses [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki){target=_blank}, whereas Ethereum uses [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki){target=_blank} or [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki){target=_blank}.
+
+如果您选择**Import from Polkadot.{js}**（从Polkadot.{js}导入），您需要确保该账号是通过私钥在Polkadot.js中创建。如果该账户是通过助记词创建，并且您尝试将其导入SubWallet，则会使用不同的公共地址。这是因为Polkadot.js使用的是[BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki){target=_blank}，而以太坊使用的是_blank}, whereas Ethereum uses [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki){target=_blank}或[BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki){target=_blank}。
```
Comment (wuzhong-papermoon) [inline tokens/connect/subwallet.md:146]:
```suggestion
如果您选择**Import from Polkadot.{js}**（从Polkadot.{js}导入），您需要确保该账号是通过私钥在Polkadot.js中创建的。如果该账户是通过助记词创建并且尝试将其导入SubWallet，您将会得到一个一样的公共地址。这是因为Polkadot.js使用的是[BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki){target=_blank}，而以太坊使用的是[BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki){target=_blank}或[BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki){target=_blank}。
```

File: tokens/connect/subwallet.md [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+
+To copy your address, click on the **Get address** icon.
+
+复制地址，点击**Get address**图标。
+
+![The tokens screen on the SubWallet browser extension.](/images/tokens/connect/subwallet/subwallet-15.png)
+
+If you have multiple accounts and have selected **All accounts** from the account dropdown menu, you'll need to select the receiving account you want to send the assets to. Otherwise, make sure that the account you're connected to (which is displayed at the top of the screen) is the account you want to send the assets to. **This should be your Moonbeam account, which is an Ethereum-style address.**
+
+如果您有多个账户，并且从账户下拉菜单中选择了**All accounts**，您需要选择用于接收资产的地址。否则，请确保您连接的账户（显示在页面顶部）是要接收资产的地址。**该账户是Moonbeam账户，即以太坊格式的地址。**
```
Comment (wuzhong-papermoon) [inline tokens/connect/subwallet.md:279]:
```suggestion
如果您有多个账户，并且从账户下拉菜单中选择了**All accounts**，您最好手动再次选择用于接收资产的地址。否则，请确认您当前连接的账户（显示在页面顶部）是正确的接收资产地址。**该账户是Moonbeam账户，即以太坊格式的地址。**
```

**Review by wuzhong-papermoon — COMMENTED** [review]
Just some minor changes for readability improvement.

File: tokens/connect/subwallet.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+本教程将引导您如何从安装SubWallet到设置钱包，以及如何将其连接至Moonbeam并发送资金。
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
+
+## 安装SubWallet {: #install-subwallet }
+
+与SubWallet交互有多种方式，例如**通过浏览器扩展程序、移动端App或网页端数据面板**。
+
+首先前往[SubWallet下载页面](https://www.subwallet.app/download.html){target=_blank}，根据个人使用习惯下载SubWallet。
```
Comment (themacexpert) [inline tokens/connect/subwallet.md:22]:
```suggestion
首先前往[SubWallet下载页面](https://www.subwallet.app/zh/download.html?lang=1){target=_blank}，根据个人使用习惯下载SubWallet。
```

Comment (themacexpert) [reply tokens/connect/subwallet.md:22]: Chinese language site

File: tokens/connect/subwallet.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
+---
+title: 如何连接SubWallet至Moonbeam
+description: 通过本教程学习如何将SubWallet（一款波卡、Substrate和以太坊综合性钱包）连接至Moonbeam。
+---
+
+# 使用SubWallet与Moonbeam交互
+
+## 概览 {: #introduction }
+
+Moonbeam的开发者和用户在使用钱包时有多种选择。得益于Moonbeam与以太坊的无缝兼容性，Moonbeam支持[SubWallet](https://www.subwallet.app/){target=_blank}等多种常用钱包。
```
Comment (themacexpert) [inline tokens/connect/subwallet.md:10]:
```suggestion
Moonbeam的开发者和用户在使用钱包时有多种选择。得益于Moonbeam与以太坊的无缝兼容性，Moonbeam支持[SubWallet](https://www.subwallet.app/zh/?lang=1){target=_blank}等多种常用钱包。
```

**Review by themacexpert — APPROVED** [review]
LGTM 👍

**Review by wuzhong-papermoon — COMMENTED** [review]
LGTM!

**Pull request merged**